### PR TITLE
[MM-770]: Added webapp testcase for create issue modal

### PR DESCRIPTION
**Pull request by Kshitij-Katiyar**
### Summary
Added webapp testcase for create issue modal

### CODE REVIEW

File: webapp/src/components/modals/create_issue/create_issue.test.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    getErrorMessage: jest.fn(() => 'Error occurred'),
+}));
+
+describe('CreateIssueModal', () => {
+    const defaultProps = {
+        close: jest.fn(),
+        create: jest.fn(() => Promise.resolve({})),
+        post: null,
+        title: '',
+        channelId: '',
```
Comment (raghavaggarwal2308) [inline webapp/src/components/modals/create_issue/create_issue.test.jsx]: Do we need to initialize these

File: webapp/src/components/modals/create_issue/create_issue.test.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        theme: {
```
Comment (raghavaggarwal2308) [inline webapp/src/components/modals/create_issue/create_issue.test.jsx:15]: I think we should also test the theme changes if possible

**Review by raghavaggarwal2308 — CHANGES_REQUESTED** [review]
Get it reviewed from a FE dev

**Comment by Kshitij-Katiyar** [comment]
> Get it reviewed from a FE dev

Hey @avas27JTG, can you please review these testcases.

File: webapp/src/components/modals/create_issue/create_issue.test.jsx (hunk 2 above)
Comment (Kshitij-Katiyar) [reply webapp/src/components/modals/create_issue/create_issue.test.jsx:15]: I tried adding them but was unable to do so

**Review by raghavaggarwal2308 — APPROVED** [review]

**Comment by Kshitij-Katiyar** [comment]
Created a PR with all these changes on MM remote, hence closing this one

**Pull request closed without merging**